### PR TITLE
Corrected some configuration and supported dynamic IP for Softlayer

### PR DESCRIPTION
**Pull request by mattcui**
Corrected some configuration in cpi.yml and added a new cpi-dynamic.yml to support to deploy a BOSH director with dynamic IP on Softlayer. Please help review and merge. Thanks.

/cc @maximilien @cppforlife

### CODE REVIEW

File: softlayer/cpi.yml [hunk 1]
```diff
@@ -84,6 +85,12 @@
   value: &softlayer
     username: ((sl_username))
     apiKey: ((sl_api_key))
+    featureOptions:
```
Comment (cppforlife) [inline softlayer/cpi.yml]: arent these values default?

Comment (mattcui) [inline softlayer/cpi.yml]: @cppforlife I removed these feature options, please review again. Thanks.

**Comment by cppforlife** [comment]
@mattcui is this still all correct? can you squash it into a single commit?

**Comment by mattcui** [comment]
@cppforlife We and other guys ever deployed with this set of yml many times, it worked as expected. I think an easy way to get a clean commit history is to re-submit a PR from a clean repo/branch, I am closing this one and already resubmitted [PR 134](https://github.com/cloudfoundry/bosh-deployment/pull/134), please merge the new PR. Thanks.

**Pull request closed without merging**